docs/source/exporthumandata.rst

```diff
@@ -44,21 +46,37 @@ like in the Handbook):
 Setting up the OSF Remote
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To use OSF as a storage, you need to provide either your OSF credentials or an OSF access token.
-You can create such a token in your account settings (`Personal access token` and then `Create token`), make sure to create a `full_write` token to be able to create OSF projects and upload data to OSF.
+To use OSF as a storage, you first need to provide either your OSF credentials (username and password) or an OSF access token.
+
+If you choose to use your credentials, proceed as follows:
+
+.. code-block:: bash
+
+    export OSF_USERNAME=YOUR_USERNAME_FOR_OSF.IO
+    export OSF_PASSWORD=YOUR_PASSWORD_FOR_OSF.IO
+
+In this example, we are going to use an OSF access token instead.
+You can create such a token in your account settings (`Personal access token` and then `Create token`).
+Make sure to create a `full_write` token to be able to create OSF projects and upload data to OSF.
 
 .. code-block:: bash
 
     export OSF_TOKEN=YOUR_TOKEN_FROM_OSF.IO
 
-We are now going to use datalad to create a sibling dataset on OSF with name `osf` - this will create a new project called `OSF_PROJECT_NAME` on the OSF account associated with the OSF token in `$OSF_TOKEN`.
+We are now going to use datalad to create a sibling dataset on OSF with name `OSF_PROJECT_NAME`.
+This will create a new project called `OSF_PROJECT_NAME` on the OSF account associated with the OSF token in `$OSF_TOKEN`.
+
+Note that the ``-s NAME_OF_REMOTE>`` flag is used to specify how ``git`` internally refers to your OSF project with the name `OSF_PROJECT_NAME`.
+It would be completely fine to use `OSF_PROJECT_NAME` also as a value for the ``-s`` flag.
+
+You can later on list your remotes from the command line using the ``git remote -v`` command.
```

Comment:
ATM this names refers to the special remote, not a Git remote IIRC.

Reply:
Still listed by `git remote -v`. Just w/o any details.

Reply:
Is there a command to list special remotes that I should refer to instead?

Should I adjust the text to make it more clear that this is not a normal remote, but a special remote?